### ISSUE TITLE
Show "Sharing" menu in `read-only` mode when user is not admin

### ISSUE THREAD

**Issue by spaenleh** [graasp/graasp-builder#556]
Currently non-admin users can see the "add user by email" field.
Currently non-admin users can see the dropdown to change a users permission on the item.

All these calls fail in the backend which guaranties the permission access scheme is respected.
We should:
- not show the "add member by email" field to users without the "admin" right
- only show the user permission in "read-only" mode to users without the "admin" right

**Comment by juancarlosfarah** [comment]
@MalinSvenberg, here's another small task to get started. Let me, @spaenleh, and/or @pyphilia know if you have any questions.

**Comment by spaenleh** [comment]
Yes, the UI should reflect that fact that `readers` (users with only `read` permission) are not allowed to modify other users permissions. They should not see a select item if they can not perform the interaction.

**Comment by MalinSvenberg** [comment]
So users with `write` access should see the admin-version of the sharing menu? 

I can't find where `readers` can see the admin-version of the sharing menu, only that the ones with `write` access can. Unless there is more than one sharing menu?

**Comment by spaenleh** [comment]
Good question @MalinSvenberg ! Users with `write` should see the `readOnly` mode of the "Authorized members" and "Pending Invitations". 
They should not be able to perform actions such as "remove permission", "change permission" or "invite/share" to user by email.
@juancarlosfarah should the users with `read` be able to see the table too ? Current behavior is to show this message which i think is not very helpful:
<img width="1028" alt="Capture d’écran 2023-02-22 à 12 11 39" src="https://user-images.githubusercontent.com/39373170/220603850-126ea410-fa71-440b-a7c7-b6358d12b1da.png">
Should we:
- disable access to the sharing menu for `read`
- allow access to the menu but display it in `readOnly` mode ?
